src/microbe_stage/components/OrganelleContainer.cs

```diff
@@ -472,6 +474,15 @@ public static void CalculateOrganelleLayoutStatistics(this ref OrganelleContaine
             }
         }
 
+        public static void UpdateEngulfingData(this ref OrganelleContainer container,
```

Comment:
```suggestion
        public static void UpdateEngulfingSizeData(this ref OrganelleContainer container,
```

I think it would be better to be explicit as "data" in engulfing could relate to things like the list of engulfed objects etc.